packages/components/src/Combobox/components/ComboboxContent/ComboboxContent.tsx

```diff
@@ -34,6 +34,11 @@ interface ComboboxContentProps {
    * Placeholder text to display in the search input. Defaults to "Search".
    */
   readonly searchPlaceholder?: string;
+
+  /**
+   * The encapsulating noun for the content of the combobox. Should be pluralized.
+   */
+  readonly contentNoun?: string;
```

Comment:
not tied to this prop name - very open to suggestions!

Reply:
off the top of my head some other synonyms could be....
- topic
- subject
- noun
- theme
- concept

I kinda like topic and subject the best. wondering about if we need to prefix with `content` when it's a prop of the content component

that said, I do like noun because it's a very.... grammatical? word. to me, it heavily implies that its usage is tied to wording, to sentence structure

hmmm, what do you think about a combination of say `subjectNoun` or `topicNoun`? or is that too redundant?

also I have one more suggestion that might change how we wanna name or prefix is

Reply:
`subjectNoun` actually makes perfect sense to me - subject nouns are the subject of a sentence so saying "I'm choosing a 13% tax rate" means the subject noun would be "tax rate". Boom. Done.